firstvoices/backend/urls.py

```diff
@@ -61,6 +62,7 @@
 sites_router.register(r"word-of-the-day", WordOfTheDayView, basename="word-of-the-day")
 sites_router.register(r"songs", SongViewSet, basename="song")
 sites_router.register(r"videos", VideoViewSet, basename="video")
+sites_router.register(r"site-widgets", SiteWidgetViewSet, basename="sitewidget")
```

Comment:
I think the url here could just be widgets, since it is already nested under /sites. Good call on making the basename sitewidget though